server/bin/pull_from_bga/games.js

```diff
@@ -0,0 +1,73 @@
+const path = require('path')
+require('dotenv').config({ path: path.join(__dirname, '../../../.env') })
+const mongoose = require('mongoose')
+require('../../configs/database')
+const axios = require('axios')
+const{
+  setIntervalAsync,
+  clearIntervalAsync
+} = require('set-interval-async/dynamic')
+const Game = require('../../models/Game')
+let skip = 9; //First 9 Don't have name entries...
```

Comment:
Should just filter all results that don't have a name at line 33

Reply:
The schema requires a unique value for the name field.  I don't think it would save it if it didn't have a name.  Just didn't know if it would crash if name was missing.

But if it works like duplicate names it'll skip those entries automatically.